ext/SymbolicsGroebnerExt.jl

```diff
@@ -70,4 +70,97 @@ function Symbolics.is_groebner_basis(polynomials::Vector{Num}; kwargs...)
     Groebner.isgroebner(polynoms; kwargs...)
 end
 
+function Symbolics.solve_multivar(eqs::Vector{Num}, vars::Vector{Num}, mult=false)
+    # do the trick
+    new_var = (@variables HAT)[1]
+
+    old_len = length(vars)
+    push!(vars, new_var)
+
+    new_eqs = []
+    generating = true
+
+    while generating
+        new_eqs = deepcopy(eqs)
+        eq = new_var
+        for i = 1:(old_len)
+            eq -= rand(1:10)*vars[i]
+        end
+        push!(new_eqs, eq)
+        new_eqs = convert(Vector{Any}, Symbolics.groebner_basis(new_eqs, ordering=Lex(vars)))
+
+        if length(new_eqs) <= length(vars) 
+            generating &= false
+        end
+
+        for i  in eachindex(new_eqs)[2:end]
+            generating |= all(Symbolics.degree(var) > 1 for var in Symbolics.get_variables(new_eqs[i]))
+        end
+    end
+
+    solutions = []
+
+    # handle "unsolvable" cases
+    if isequal(1, new_eqs[1])
+        return solutions
+    end
+    if length(new_eqs) < length(vars)
+        throw("Infinite number of solutions")
```

Comment:
Btw, should this check go before the loop above ?

Reply:
These checks are made after the basis is computed, which is done inside the loop. One curious thing is the case of infinite number of solutions, 
```julia
for ...
        if length(new_eqs) <= length(vars) 
            generating &= false
        end
end
```
I guess this loop hangs infinitely for the infinite equation case?

Reply:
```julia
julia> eqs = [x+y+z, z*x*y, z+3x+y]
3-element Vector{Num}:
  x + y + z
      x*y*z
 3x + y + z

julia> Symbolics.solve(eqs, [x,y,z])
ERROR: "Infinite number of solutions"
```
Nevermind